src/app/components/context-menu/context-menu.component.ts

```diff
@@ -167,9 +167,8 @@ class ExecutingCut {
             undefined
         ) {
             this.feedbackService.showMessage(
-                'No arc selected via the drawing area!',
+                "Please select arcs in a DFG by click or clicked-hover to partition it's activities and thus define a cut first.",
```

Comment:
Würde es eher so schreiben.
```suggestion
               "Please select arcs in a DFG, by 'clicking' or 'click-hovering', to partition it's activities and thus define a cut first.",
```